packages/engine/src/scene/systems/EnvironmentSystem.tsx

```diff
@@ -44,6 +45,8 @@ const EnvmapReactor = (props: { backgroundEntity: Entity }) => {
   const groupComponent = useComponent(entity, GroupComponent)
 
   useEffect(() => {
+    // TODO use spatial queries
+    if (!haveCommonAncestor(entity, props.backgroundEntity)) return
```

Comment:
What bug was this fixing?

Reply:
I forgot to include this in a previous PR. This ensures that the skybox only affects the envmaps of the entities in the scene it is loaded in, essentially preventing disparate scenes from affecting eachother